common/src/main/scala/gu/msnotifications/Responses.scala

```diff
@@ -6,17 +6,17 @@ import models.WindowsMobile
 import notifications.providers.{RegistrationResponse => RegistrarResponse, UserIdNotInTags}
 import org.joda.time.DateTime
 import play.api.libs.ws.WSResponse
-
-import scala.util.Try
-import scalaz.\/
+import scala.util.{Failure, Success, Try}
+import scala.xml.Elem
+import scalaz.{-\/, \/-, \/}
```

Comment:
it was mentioned in previous PRs by @davidfurey - `import scalaz.syntax.either._` gives opportunity to convert:
`\/-(Some(dateTime))` to `Some(dateTime).right` - IMHO its more readable.